gtsfm/multi_view_optimizer.py

```diff
@@ -52,7 +50,7 @@ def create_computation_graph(
         v_corr_idxs_graph: Dict[Tuple[int, int], Delayed],
         intrinsics_graph: List[Delayed],
         gt_poses_graph: List[Delayed] = None,
-    ) -> Tuple[Delayed, Delayed, Optional[Delayed], Optional[Delayed]]:
+    ) -> Tuple[Delayed, Delayed, Optional[Delayed]]:
```

Comment:
docstring needs to be updated

Reply:
why is the last return arg marked as Optional?

Reply:
i made it not optional, feel free to change back later if you disagree